CHANGELOG.md

```diff
@@ -50,4 +52,5 @@ Each section shall contain a list of action items of the following format: `<bri
 - Replaced high-lighting from "Closed Issues" section with correct list ([#139](https://github.com/IHE/DEV.SDPi/issues/139)).
 - Remove leading dashes from table and figure captions if the section number is empty ([#144](https://github.com/IHE/DEV.SDPi/issues/144)).
 - Reviewed all "Verson Note" and version-specific (i.e, 1.0) content and either reworded or removed ([#148](https://github.com/IHE/DEV.SDPi/issues/148))
-- Updated all references to the sdpi-fhir Github ([#162](https://github.com/IHE/DEV.SDPi/issues/162))
\ No newline at end of file
+- Updated all references to the sdpi-fhir Github ([#162](https://github.com/IHE/DEV.SDPi/issues/162))
+- Addressed editorial issues from the IHE Publications review ([#189](https://github.com/IHE/DEV.SDPi/issues/189))
```

Comment:
This needs to go to the editorial fixes section above. This one is the 1.0.1 editorial fixes section.